Library/Homebrew/build.rb

```diff
@@ -81,6 +82,7 @@ def install
     if superenv?
       ENV.keg_only_deps = keg_only_deps
       ENV.deps = formula_deps
+      ENV.run_time_deps = run_time_deps
       ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) }
```

Comment:
It's used to point to paths to in the XQuartz SDK, which shouldn't be necessary for `xorg` because it's in the brew prefix.